### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

File: Documentation/core-repos.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# .NET Core Repos
+
+There are many repos that make up .NET Core. To file an issue, make a PR, or engage on specific topics, it is best to engage with developers working on the right repo. A largely complete set .NET Core repositories follows.
+
+## .NET Core
+
+* [dotnet/cli](https://github.com/dotnet/cli) - .NET Core CLI tools
+* [dotnet/corefx](https://github.com/dotnet/corefx) - .NET Core framework APIs
```
Comment (danmosemsft) [inline Documentation/core-repos.md]: is framework lower case?

Comment (danmosemsft) [inline Documentation/core-repos.md]: Runtime is capitalized

File: Documentation/core-repos.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* [dotnet/standard](https://github.com/dotnet/standard) - .NET Standard definition
+
+## Languages
+
+* [dotnet/roslyn](https://github.com/dotnet/roslyn) - Roslyn and C# questions
+* [dotnet/netcorecli-fsc](https://github.com/dotnet/netcorecli-fsc) - F# questions and
+
+## NuGet
+
+* [nuget/home](https://github.com/nuget/home) - NuGet questions and
```
Comment (danmosemsft) [inline Documentation/core-repos.md]: .. and ... ?

File: Documentation/core-repos.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* [dotnet/sdk](https://github.com/dotnet/sdk) - .NET Core SDK (used by Visual Studio and CLI)
+
+## .NET Standard
+
+* [dotnet/standard](https://github.com/dotnet/standard) - .NET Standard definition
+
+## Languages
+
+* [dotnet/roslyn](https://github.com/dotnet/roslyn) - Roslyn and C# questions
+* [dotnet/netcorecli-fsc](https://github.com/dotnet/netcorecli-fsc) - F# questions and
```
Comment (danmosemsft) [inline Documentation/core-repos.md]: .. .and ?

File: Documentation/core-repos.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+* [dotnet/docs](https://github.com/dotnet/docs) - .NET documentation
+* [dotnet/sdk](https://github.com/dotnet/sdk) - .NET Core SDK (used by Visual Studio and CLI)
+
+## .NET Standard
+
+* [dotnet/standard](https://github.com/dotnet/standard) - .NET Standard definition
+
+## Languages
+
+* [dotnet/roslyn](https://github.com/dotnet/roslyn) - Roslyn and C# questions
```
Comment (danmosemsft) [inline Documentation/core-repos.md]: add also https://github.com/dotnet/csharplang

File: README.md [hunk 5]
```diff
@@ -1,67 +1,41 @@
...
-> * [dotnet/sdk](https://github.com/dotnet/sdk) - for .NET Core SDK (used by Visual Studio and CLI) issues and questions
-> * [dotnet/standard](https://github.com/dotnet/standard) - for .NET Standard issues and questions
-> * [nuget/home](https://github.com/nuget/home) - for NuGet questions and issues
-> * [aspnet/home](https://github.com/aspnet/home) - for ASP.NET Core questions and issues.
-> * [aspnet/Docs](https://github.com/aspnet/Docs) - for ASP.NET and ASP.NET Core documentation issues and questions
-> * [dotnet/netcorecli-fsc](https://github.com/dotnet/netcorecli-fsc) - for F# questions and issues
-
-> 
-> We will be removing the ability to file issues on this repo in the near future.
+The dotnet/core repository a good starting point for .NET Core.
```
Comment (danmosemsft) [inline README.md]: .. is a..

File: README.md [hunk 6]
```diff
@@ -71,35 +45,3 @@ Some projects license documentation and other forms of content under
 [Creative Commons Attribution 4.0](http://creativecommons.org/licenses/by/4.0/).
 
 See specific projects to understand the license used.
-
-## Understanding the relationship between .NET Core and the .NET Framework
```
Comment (danmosemsft) [inline README.md]: was none of this useful? if it's somewhere else maybe link to it?

Comment (richlander) [inline README.md]: Good q but it's all in docs in one place or another. We could link to specific docs if you found that content compelling.

It was useful. I wrote it all ;)

File: Documentation/core-repos.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# .NET Core Repos
+
+There are many repos that make up .NET Core. To file an issue, make a PR, or engage on specific topics, it is best to engage with developers working on the right repo. A largely complete set .NET Core repositories follows.
```
Comment (mairaw) [inline Documentation/core-repos.md]: "To file an issue, make a PR, or engage on specific topics, it is best to engage with developers working on the right repo. A largely complete set .NET Core repositories follows." is a bit hard to read.
Suggestion:
"Different individuals (or teams) track different repositories. So, you'll get better results if you file an issue, make a PR, or engage on specific topics on the repo specific to that work. The following list shows the main .NET Core repositories:"

File: Documentation/core-repos.md (hunk 1 above)
Comment (mairaw) [inline Documentation/core-repos.md]: .NET Core framework could be confusing to folks IMO

File: README.md [hunk 8]
```diff
@@ -1,67 +1,42 @@
...
+- [Get started with .NET Core](https://www.microsoft.com/net/core)
+- [Download .NET Core](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md)
 
-## .NET Core Platform
+## Learn about .NET Core 
 
-The .NET Core platform is made of several components, which includes the
-managed compilers, the runtime, the base class libraries, and numerous application models such as
-ASP.NET.
+- [Learn about .NET Core]
```
Comment (mairaw) [inline README.md]: url should be on the same line, also should it have a different link text than the section title?

File: README.md [hunk 9]
```diff
@@ -1,67 +1,42 @@
...
-If you're new to .NET Core and have 10 minutes to try it, start here: 
-- [What is .NET Core?](https://www.microsoft.com/net/core/platform)
-- [Get Started with the Official releases of .NET Core on Windows, OSX and Linux](https://www.microsoft.com/net/core)
+If you're new to .NET Core, you can get started in 10 minutes: 
 
-If you have some more time and want to go deeper or get the latest builds:
-- [Get the latest builds of .NET Core on Windows, OSX and Linux](https://github.com/dotnet/core-setup/blob/master/README.md)
-- [Get the latest builds of ASP.NET Core on Windows, OSX and Linux](https://github.com/aspnet/home)
+- [Get started with .NET Core](https://www.microsoft.com/net/core)
+- [Download .NET Core](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md)
```
Comment (mairaw) [inline README.md]: is this the best link for newbies to download .NET Core?

File: README.md [hunk 10]
```diff
@@ -1,67 +1,42 @@
...
-* [Roadmap](roadmap.md)
-* [.NET Core Framework](https://github.com/dotnet/corefx)
-* [.NET Core Runtime](https://github.com/dotnet/coreclr)
-* [.NET Compiler Platform ("Roslyn")](https://github.com/dotnet/roslyn)
-* [ASP.NET Core](https://github.com/aspnet/home)
+## Getting help
+
+- [File a general .NET Core issue](https://github.com/dotnet/core/issues)
+- [File a more specific issue](Documentation/core-repos.md)
+- [Ask on StackOverflow](https://stackoverflow.com/questions/ask)
```
Comment (mairaw) [inline README.md]: Stack Overflow (with a space)

File: README.md [hunk 11]
```diff
@@ -1,67 +1,42 @@
...
-* [.NET Compiler Platform ("Roslyn")](https://github.com/dotnet/roslyn/wiki/Contributing-Code)
-* [ASP.NET Contributing Guide](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md)
+## Download .NET Core
 
-You are also encouraged to start a discussion by posting on the
-[.NET Foundation Forums](http://forums.dotnetfoundation.org/) or filing an
-issue in the corresponding GitHub project. See the contributing guides for more
-details.
+- [Released builds](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md)
+- [_latest_ .NET Core SDK](https://github.com/dotnet/cli/blob/master/README.md#installers-and-binaries)
```
Comment (mairaw) [inline README.md]: I tend to think of latest as the latest supported, not the daily build

File: README.md (hunk 9 above)
Comment (richlander) [inline README.md]: The problem is that not everyone is looking for the same version. The marketing site doesn't do a good job of catering to that. I also have plans to make the download pages/release notes easier to use.

File: Documentation/core-repos.md (hunk 7 above)
Comment (richlander) [inline Documentation/core-repos.md]: That feels like passive construction in the sense that the second sentence contains the more important verb or the first.

**Comment by richlander** [comment]
Made additional updates per feedback: https://github.com/dotnet/core/commit/40edabb3447490204d79e4bd0a9db09a08d44ae0

File: README.md [hunk 12]
```diff
@@ -1,67 +1,42 @@
...
 
-The core repository is the starting point to engage in and learn about the
-.NET Core stack. 
-
-> **Note:** please note that this repository is not for filing product issues.
-> If you run into an issue using .NET Core, there are multiple repos where you can 
-> file an issue:
-> * [dotnet/cli](https://github.com/dotnet/cli) - for CLI tools and questions
-> * [dotnet/corefx](https://github.com/dotnet/corefx) - for API issues and questions
-> * [dotnet/coreclr](https://github.com/dotnet/coreclr) - for runtime issues
```
Comment (Petermarcu) [inline README.md]: Should we add core-setup? That's where we build netcoreapp, the host, and the runtime installers.

Comment (richlander) [inline README.md]: I added a smattering of repos to get started and was going to let people (like you) add more (as you did).

**Pull request merged**